src/iOS.Core/Controllers/LoginAddViewController.cs

```diff
@@ -121,6 +179,19 @@ public override void ViewDidLoad()
             base.ViewDidLoad();
         }
 
+        private void Type_ValueChanged(object sender, EventArgs e)
+        {
+            var organizationList = _organizations.OrderBy(o => o.Name).ToList();
+            var collectionsNames = _writeableCollections.Where(c => c.OrganizationId == _organizations.ElementAt(OrganizationCell.SelectedIndex).Id).Select(s => s.Name).ToList();
+            if (collectionsNames.Count == 0)
+            {
+                collectionsNames.Insert(0, AppResources.NoCollectionsToList);
+            }
+
+            CollectionCell.Items = collectionsNames;
```

Comment:
🎨 Extract to method for reuse; given this is the same as in [above lines](https://github.com/bitwarden/mobile/pull/3265/files#diff-0ab7e04015c2d47f1b83c22c3f9128358377c95c1caecf8f34a416f5eec7a83fR150-R156)